packages/generate-bot/test/generate-bot.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * Copyright 2019 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// define types for a few modules used by probot that do not have their
+// own definitions published. Before taking this step, folks should first
+// check whether type bindings are already published.
+
+const GenerateBot = require("../main.js");
+const { expect } = require("chai");
+const fs = require("fs");
+const recursive = require("recursive-readdir");
+const rimraf = require("rimraf");
+const snapshot = require("snap-shot-it");
+const path = require("path");
+
+const readAllFiles = function(dirNameRead, contentString) {
+  const files = fs.readdirSync(dirNameRead);
+  contentString = contentString || null;
+  files.forEach(function(file) {
+    const readName = path.join(dirNameRead, file);
+    if (fs.statSync(readName).isDirectory()) {
+      readAllFiles(readName, contentString);
+    } else {
+      contentString += fs.readFileSync(readName);
+    }
+  });
+  console.log(contentString);
```

Comment:
let's get rid of this `console.log`, because it makes tests hard to debug (it's causing a lot of output).